src/MessageHandler/Organization/AddPackageHandler.php

```diff
@@ -29,7 +29,8 @@ public function __invoke(AddPackage $message): void
                     $message->type(),
                     $message->url(),
                     $message->metadata(),
-                    $message->keepLastReleases()
+                    $message->keepLastReleases(),
+                    $message->hasSecurityScanEnabled()
```

Comment:
You have the organization here in the handler, you do not have to pass this feature through a message object.
```php
$organization = $this->organization->getById(Uuid::fromString($message->organizationId()));
$organization->addPackage(new Package(..., $organization->hasEnabledSecurityScan()));
```